parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/BytePacker.java

```diff
@@ -105,4 +116,16 @@ public void unpack8Values(final byte[] input, final int inPos, final int[] outpu
   public void unpack32Values(byte[] input, int inPos, int[] output, int outPos) {
     unpack32Values(ByteBuffer.wrap(input), inPos, output, outPos);
   }
+
+  public void unpackValuesVector(final byte[] input, final int inPos, final int[] output, final int outPos) {
```

Comment:
Please add appropriate comments over newly added definitions, like the ones over top over scalar routines.